prdoc/pr_3532.prdoc

```diff
@@ -0,0 +1,13 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Remove deprecated `trait Store`
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      The deprecated `trait Store` feature has been removed from the codebase.
```

Comment:
Please also mention what this means for pallet devs, and what they need to update in their pallets to make it compile again.

Reply:
got it, will do